README.md

```diff
@@ -219,11 +192,20 @@ or [Google Play Store](https://play.google.com/store/apps/details?id=com.samsung
 log in with the same email address and password used for your developer workspace account, and 
 create a location (if you have not already done so)
 
-Put the SmartThings mobile app in [developer mode](https://smartthings.developer.samsung.com/docs/guides/testing/developer-mode.html) and tap the "+" button at the top to
-add a device. Select the _Partner devices_ option, and you should see a _My Testing Devices_ menu item. 
+Put the SmartThings mobile app in [developer mode](https://developer.smartthings.com/docs/devices/cloud-connected/test-your-connector#enable-developer-mode-in-the-smartthings-app) 
+and tap the "+" button at the top to add a device. Select the _Partner devices_ option, and you should see a _My Testing Devices_ menu item. 
 Tap on it and you should see your connector name. Tap on it to install your connector, using the same email
```

Comment:
The connector will not appear in My Testing Devices when it is created with the CLI. Invitations are the only way to install it. This is somewhat of a problem because that means it can't actually be tested in the mobile app, but that's the way it is for now

Reply:
@bflorian will the connector appear in the SmartThings app after accepting an invitation? E.g. the process for installing the connector and testing in the app would be: 

1. Create an invitation to send to yourself
2. Accept the invitation and install to an existing location on your account
3. Connector will now appear in your SmartThings app, no developer mode required